README.md

```diff
@@ -175,6 +176,11 @@ To add resources or to contribute read the [Contribution-guide](https://github.c
 - :hammer_and_wrench: [Pixabay](https://pixabay.com/)
 - :books: [UX Planet](https://uxplanet.org/)
 
+---
+
+## Docker
+- :video_camera: [kickstart docker](https://www.youtube.com/watch?v=gAkwW2tuIqE)
+ 
```

Comment:
here's one more useful resource for learning Docker
https://www.youtube.com/watch?v=pTFZFxd4hOI
quick add:
- :video_camera: [Complete Docker Tutorial](https://www.youtube.com/watch?v=pTFZFxd4hOI)